src/SourceGenerators/Uno.UI.SourceGenerators.Tests/XamlCodeGeneratorTests/Out/TIM/XamlCodeGenerator_MainPage_d6cd66944958ced0c513e0a04797b51d.cs

```diff
@@ -251,32 +289,20 @@ void IMainPage_Bindings.Update()
 			{
 				var owner = Owner;
 				owner._component_0.ApplyXBind();
-				owner.__0_Click_P1_Button_Click_Builder?.Invoke( owner._component_0);
-				owner.__0_Click_P1_Button_Click_Builder = null;
-				owner.__1_Click_P2_Button_Click_Builder?.Invoke( owner._component_1);
-				owner.__1_Click_P2_Button_Click_Builder = null;
-				owner.__2_Click_P3_Button_Click_Builder?.Invoke( owner._component_2);
-				owner.__2_Click_P3_Button_Click_Builder = null;
+				owner.__ApplyMethod_0_Click_Initialize();
+				owner.__ApplyMethod_1_Click_Initialize();
+				owner.__ApplyMethod_2_Click_Initialize();
 				owner._component_1.ApplyXBind();
-				owner.__0_Click_P1_Button_Click_Builder?.Invoke( owner._component_0);
-				owner.__0_Click_P1_Button_Click_Builder = null;
-				owner.__1_Click_P2_Button_Click_Builder?.Invoke( owner._component_1);
-				owner.__1_Click_P2_Button_Click_Builder = null;
-				owner.__2_Click_P3_Button_Click_Builder?.Invoke( owner._component_2);
-				owner.__2_Click_P3_Button_Click_Builder = null;
+				owner.__ApplyMethod_0_Click_Initialize();
```

Comment:
We should not call those multiple times per component.

Reply:
There is a flag to execute it only once https://github.com/unoplatform/uno/pull/19418/files#diff-44fef35d54c0e184916e1b413012dc9653c86412cbb8cd629132e1d4fd931bd8R138